nginx/nginx.conf.d/coala.io.conf

```diff
@@ -1,77 +1,70 @@
 server {
     listen 443 ssl;
     server_name coala.io www.coala.io;
-	
+
     location / {
         proxy_pass             http://192.30.252.153;
         proxy_set_header       Host coala.io;
         proxy_set_header       X-Real-IP $remote_addr;
         proxy_intercept_errors on;
     }
 
-    # URL shortener
-    location /newcomer  { return 302 http://api.coala.io/en/latest/Developers/Newcomers_Guide.html; }
```

Comment:
nginx doesnt support redirects without a protocol?

if not, can you add a PR first to replace http->https or https->http , so then this diff has only minimal changes.

Reply:
it's not that nginx doesn't support it. It just throws HTTP 302 with whatever after it. Some programs get confused when there's no protocol.

Reply:
I think I'll just make some changes to the `coala.io.conf` so this diff will be lower.

Reply:
This hasnt been done , it appears